demo/atom/image/playground

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react/react-in-jsx-scope, react/jsx-no-undef */
+
+const listImageStyles = {
+  display: 'flex',
+  flexWrap: 'wrap'
+}
+
+const containerImageStyles = {
+  border: '2px solid #ccc',
+  height: '200px'
+}
+
+const listItem = {
+  width: '300px',
+  margin: '10px',
+  display: 'flex',
+  flexDirection: 'column'
+}
+
+const Spinner = ({ className }) => {
+  const spinnerStyles = {
+    width: '100px',
+    zIndex: 99
+  }
+
+  return (
+    <div className={className}>
+      <img
+        style={spinnerStyles}
+        src="https://zippy.gfycat.com/SkinnySeveralAsianlion.gif"
+        alt=""
+      />
+    </div>
+  )
+}
+
+const IMAGES = {
+  FINAL: 'https://picsum.photos/4000?image=980',
+  PLACEHOLDER: 'https://picsum.photos/50?image=980',
+  SKELETON:
+    'https://cdn1.iconfinder.com/data/icons/online-wireframes/32/Wireframe_Photo_Album_Picture-256.png',
+  BAD: 'https://pic__sum.pho__tos/50?image=980'
+}
+
+return (
+  <div>
+    <h1>AtomImage</h1>
+    <ul style={listImageStyles}>
+      <li style={listItem}>
+        <div style={containerImageStyles}>
+          <AtomImage
+            style={{ border: '1px solid' }}
```

Comment:
Do we really leave the door open to do this? Not sure if it's a good practice to leave "free" props opened to the public API of the component (regarding this line https://github.com/SUI-Components/sui-components/blob/8c3b74012d467547e0110ad19e53811dddec463c/components/atom/image/src/index.js#L66).

A part from this, I think it's a very nice feature we got with this component 😉 .